### PR TITLE
Fixes attacking blobs that should be unreachable behind other blobs

### PR DESCRIPTION
**Pull request by Razharas**
Fixes attacking blobs diagonally behind other blobs
Fix https://github.com/tgstation/-tg-station/issues/2664

### CODE REVIEW

File: code/game/gamemodes/blob/theblob.dm [hunk 1]
```diff
@@ -45,6 +45,20 @@
 	return ..()
 
 
+/obj/effect/blob/Adjacent(var/atom/neighbour)
+	. = ..()
+	if(.)
+		var/list/result = list()
+		var/direction = get_dir(src, neighbour)
+		var/list/dirs = list("9" = list(NORTH, WEST), "5" = list(NORTH, EAST), "6" = list(SOUTH, EAST), "10" = list(SOUTH, WEST))
```
Comment (RemieRichards) [inline code/game/gamemodes/blob/theblob.dm]: swap all of these strings for their values, ```"9"``` is just ```"[NORTHWEST]"```, etc.

**Comment by Shapsy** [comment]
i feel like this is gonna make fighting blobs more tedious but eh

**Comment by Razharas** [comment]
Its a 2 years old bug thus itll be fixed no matter the cost

**Comment by KorPhaeron** [comment]
So can you not hit blobs diagonally at all?

**Comment by TechnoAlchemisto** [comment]
attacking blobs diagonally is kind of a big deal when it comes to fighting blobs.

**Comment by ChangelingRain** [comment]
it appears to only specifically prevent attacking blobs that are hidden behind other blobs, see the linked issue

**Comment by Razharas** [comment]
>People not reading the original issue

**Comment by PKPenguin321** [comment]
yeah guys you should still be able to attack blobs diagonally like this, just not if they're behind two other blobs. look at the issue raz linked.

**Pull request merged**